CONTRIBUTING.md

```diff
@@ -0,0 +1,466 @@
+  starting point for top-down linkage.
+
+- Bottom-up linkage: Add links from more specific documents to more general documents to help
+  readers understand the full context of the topics being discussed. Adding links from
+  module, class, or protocol documentation comments to conceptual documentation overviews can be
+  particularly effective.
+
+- Sideways linkage: Add links to documents on subjects that help readers better understand the
+  content of your document.
+
+---
+
+## Documentation Types
+Documentation is an important part of any product or feature because it lets users know how to
+properly use a feature that has been implemented. These guidelines are meant to be a quick and easy
+reference for types of documentation. For information on documentation style guidelines, see
+[Documentation Style Guide](#documentation-style).
+
+### Conceptual, procedural, or reference documentation
+
+Most documentation can be divided into these categories:
+
+- [Reference](#reference-documentation) - Documentation that provides a source of information about
+  parts of a system such as API parameters.
+
+- [Conceptual](#conceptual-documentation) - Documentation that helps you understand a concept such
+  as the Substrate runtime.
+
+- [Procedural](#procedural-documentation)
+
+    - How-to - Documentation that provides steps on how to accomplish a goal such as setting up your computer to compile Substrate.
+
+    - Tutorial - Documentation that provides steps of a learning path (this tends to be a much bigger
+      procedure than a how-to) such as create a Substrate runtime module.
+
+**You should write a reference document** if you need to provide information about parts of a system
+including, but not limited to APIs and CLIs. Reference documentation should allow the user to
+understand how to use a specific feature quickly and easily.
+
+**You should write a conceptual document** if you plan on explaining a concept about a product.
+Conceptual documents explain a specific concept, but for the most part they do not include actual
+examples. They provide essential facts, background, and diagrams to help your readers build a
+foundational understanding of a product or topic. You should not explain industry standards that
+your audience should be familiar with, for example, TCP/IP. You might explain how this concept ties
+in with your feature, but you should not explain the basics behind that industry-standard concept.
+
+**You should write a procedural document** if you plan on explaining to a user how to use a specific
+feature and are able to guide a user through simple numbered steps. Procedural documents tend to
+reinforce the concepts that were explained in a conceptual document by giving one or more
+examples that might be useful for users.
+
+Procedural documents are divided into two categories:
+
+- **How-to** - Consider writing a how-to when you want to help the user accomplish a very specific
```

Comment:
could I just add recipes in here somehow? like I said further above, I think the mapping from these categorizations to our existing workflow should be more explicit

something like "If the how-to pertains to runtime/module development patterns, it probably belongs in the recipes." Idk it feels out of place because there is no clear references to our doc workflow in this section but I'm saying maybe there should be.

Reply:
I think we should keep this content free of the implementation details.  The Substrate Recipes is a collection of "how-to"s